marlowe-playground-client/src/Marlowe/Monaco.js

```diff
@@ -0,0 +1,60 @@
+/*eslint-env node*/
+'use strict';
+
+const monaco = require('src/Marlowe/Monaco.ts');
+
+exports.completionItemProvider_ = function (suggestionsProvider) {
+    let m = require('src/Marlowe/Monaco.ts');
+    let uncurriedSuggestions = (stripParens, contract, range) => suggestionsProvider(stripParens)(contract)(range);
+    return new m.MarloweCompletionItemProvider(uncurriedSuggestions);
+}
+
+exports.codeActionProvider_ = function (provideCodeActions) {
+    let m = require('src/Marlowe/Monaco.ts');
+    let uncurriedProvideCodeActions = (a, b) => provideCodeActions(a)(b);
+    return new m.MarloweCodeActionProvider(uncurriedProvideCodeActions);
+}
+
+exports.documentFormattingEditProvider_ = function (format) {
+    let m = require('src/Marlowe/Monaco.ts');
```

Comment:
How come you re-required the same file several times? Can't you reuse the `const monaco` definition?

(If you actually can't, I think it needs a code-comment.)

Reply:
old artefact, deleting.